src/components/Footer.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { FaArrowRight} from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { GrMailOption } from 'react-icons/gr';
+import { IconContext } from "react-icons";
+
+import {
+  AiFillLinkedin,
+  AiFillTwitterSquare,
+  AiFillFacebook,
+  AiOutlineWhatsApp
+} from 'react-icons/ai';
+import { nameLogo } from '../assets';
+
+const Footer = () => {
+  const [email, setEmail] = useState('');
+  return (
+    <div className='footer'>
+      {/* footer logo ______________  */}
+      <div className='footer-logo'>
+        <img src={nameLogo} alt='logo' />
+      </div>
+
+      <div className='footer-contact'>
+        <h2>Office</h2>
+        <ul>
+          <li>KG 11 Av</li>
```

Comment:
You can please map through those address items, and have them as constants in the constants/index.js file